docs/TRACES.md

```
@@ -0,0 +1,474 @@
+# Available Spans
+
+This document lists all available spans in Amaru, auto-generated from the code.
+
+For information on how to use and filter these spans, see [monitoring/README.md](../monitoring/README.md).
+
+
+### target: `consensus::chain_sync`
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)